example/ipfragreass/odp_ipfragreass.c

```diff
@@ -230,7 +230,7 @@ int main(void)
 	odp_pool_t fragment_pool;
 	odp_shm_t shm;
 	odp_cpumask_t cpumask;
-	odph_odpthread_t threads[MAX_WORKERS] = {};
+	odph_odpthread_t threads[MAX_WORKERS];
```

Comment:
What is the added value of removing this warning? It is rather overly paranoid.

Reply:
These also resulted -Wmissing-field-initializers